cni/cmd/istio-cni-repair/main.go

```diff
@@ -166,6 +176,12 @@ func logCurrentOptions(bpr *repair.BrokenPodReconciler, options *ControllerOptio
 	}
 }
 
+func init() {
+	metrics.PodsRepaired = monitoring.NewSum("istio_cni_repair_pods_repaired_total",
```

Comment:
super nit: do we need `repair` in the name twice?  would something like: `istio_cni_repaired_pods_total` be just as clear?

fwiw, `repaired_pods` is slightly preferable to `pods_repaired` to my ears. But I'm biased by https://cloud.google.com/apis/design/naming_convention.

Reply:
I was treating `istio_cni_repair` as a prefix, so that they are all in a block and someone quickly scanning them can find all the repair-related metrics if we later introduce other cni metrics.

Not particularly tied to that, but that was my rationale.